### ISSUE TITLE
Environment Variables are always sent to Raygun

### ISSUE THREAD

**Issue by t0rr3sp3dr0** [MindscapeHQ/raygun4py#85]
Currently is not possible to prevent environment variables from being sent to Raygun's servers. It is common to have sensitive information like API keys on environment variables, and not have a way to disable this behavior could lead to a serious security breach via Raygun.

This problem doesn't exist on libs for other languages like `raygun4java` and `raygun4ruby`.

**Comment by brock** [comment]
Have you added `filtered_keys: []` to the config when you're setting up Raygun in your python app? They come through like this when you've added them (I'm a customer who uses this feature, not a Raygun employee):

```
KEY: "<filtered>"
SECRET: "<filtered>"
```

Not sure if you can specify it like this:

```
filtered_keys: ['*']
```

but it might be worth a try

**Comment by t0rr3sp3dr0** [comment]
Unfortunately there is no wildcard for `filtered_keys`, and it is unfeasible to list all forbidden keys on my use-case.

**Comment by t0rr3sp3dr0** [comment]
I have identified another problem caused by this behavior, some services that were in production for more than 1 year were just not logging anything on Raygun. There were too many environment, global, and local variables being sent to Raygun, that silently caused error 413 on API, no exception was raised when this happened.

**Comment by UberMouse** [comment]
Hi @t0rr3sp3dr0,

Sorry you're running into this problem! We'll take a look. Just to confirm, what you're after here is the ability to wildcard ignore **all** your environment variables?

I see in your most recent comment you mention global/local variables but we do already support turning transmission of those off if that is a problem. Is there any other functionality that would be desireable there?

Regards,
Taylor

**Comment by t0rr3sp3dr0** [comment]
@UberMouse, that was exactly what I was looking for!

My last comment concerns the fact that Raygun sends global, local, and environment variables by default, what would cause error 413 in many scenarios. I think it would be more desirable in these cases that the variables would not be sent by default and that users would enable this option manually.

But this would be a problem for another issue and your PR solves this issue. 😁

**Comment by UberMouse** [comment]
I've just released v4.3.0 of Raygun4Py with a new config option, `transmit_environment_variables` to allow you to solve this @t0rr3sp3dr0
